Makefile

```diff
@@ -9,7 +9,7 @@ package_name=$(app_name)
 cert_dir=$(HOME)/.nextcloud/certificates
 github_account=nextcloud
 branch=master
-version+=0.6.3
+version+=0.6.4
```

Comment:
actually I would put this to `master` or something.
You can still build 0.6.4 then with `make release version=0.6.4`

That's how we do it in talk. Makes sure temporary tests dont get confused with the final version afterwards.

Reply:
ahh, yeah makes sense!